content/developers/apps/setting-permissions-for-github-apps.md

```diff
@@ -11,10 +11,4 @@ versions:
   github-ae: '*'
 ---
 
-When you create a GitHub App, you can select the permissions it needs to access end user data. Permissions can also be added and removed. For more information, see "[Editing a GitHub App's permissions](/apps/managing-github-apps/editing-a-github-app-s-permissions/)."
-
-### Metadata permissions
```

Comment:
@georgicodes  👋🏻 Just checking whether you intentionally meant to remove this whole section?
I'm wondering if the second paragraph still applies. If you confirm that the whole section is to be removed, I just wanted to highlight that it's still being linked to in https://docs-3861--gkfix-deafult-perms.herokuapp.com/en/developers/apps/creating-a-github-app-using-url-parameters#github-app-permissions (see `metadata` entry in table), and that you may need to remove that link or the whole row (I don't know enough to make the call) in this PR, otherwise there is going to be a broken link.

Reply:
@mchammer01 hello there! Sorry I can't quite see where the broken link would be. I clicked on metadata in the table and it took me here: https://docs-3861--gkfix-deafult-perms.herokuapp.com/en/rest/reference/permissions-required-for-github-apps#metadata-permissions.

I do think it makes sense to remove this section given we don't default setting he metadata permissions anymore.

Reply:
@georgicodes 👋🏻 - you are absolutely right, and I do apologize. I think I need new glasses 😞 
I'm going to see if I can get this merged soon, thanks for confirming about the removal of the whole section 🙂